haproxy/config.go

```diff
@@ -131,3 +139,18 @@ func newHaConfig(baseDir string, sd *lib.Shutdown) (*haConfig, error) {
 
 	return cfg, nil
 }
+
+func hashPassword(password string) (string, error) {
+	crypter := crypt.SHA256.New()
+	return crypter.Generate([]byte(password), []byte("$5$salt"))
+}
+
+func createRandomString(n int) string {
+	var dictionary = []rune("_0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz")
+	b := make([]rune, n)
+	rand.Seed(time.Now().UnixNano())
+	for i := range b {
+		b[i] = dictionary[rand.Intn(len(dictionary))]
```

Comment:
This should use `crypto/rand` instead of `math/rand`. `math/rand` is not a secure random source.

Reply:
I applied suggested changes.